cmd/tempo-query/Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM jaegertracing/jaeger-query:1.57.0
```

Comment:
Could this be kept and build the image with the jaeger-query?

Reply:
And then a users has the option to run jaeger-query or tempo-query?
Y, we can do that. 👍